sale_comment_template/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO sale_order_base_comment_template_rel
+        (base_comment_template_id, sale_order_id)
+        SELECT bct.id AS base_comment_template_id, so.id AS sale_order_id
+        FROM sale_order so
+        JOIN base_comment_template bct ON bct.id = so.old_comment_template1_ids
```

Comment:
`old_comment_template1_ids` shouldn't be `comment_template1_id`? (and the same for the other)

Reply:
Anyways, you don't need the JOIN to this table, just select `so.comment_template1_id` above.

Reply:
I have fixed myself this point for not having an error on upgrade.